task/run-script-oci-ta/0.1/run-script-oci-ta.yaml

```
@@ -161,9 +167,13 @@ spec:
         echo 'root:1:4294967294' | tee -a /etc/subuid >>/etc/subgid
 
         UNSHARE_ARGS=()
+        HERMETO_MOUNT=""
+        HERMETO_SOURCE=""
         if [ "${HERMETIC}" == "true" ]; then
           echo "The script will be executed with network isolation"
           UNSHARE_ARGS+=("--net")
+          HERMETO_MOUNT="--volume /var/workdir/cachi2:/cachi2"
+          HERMETO_SOURCE="source /cachi2/cachi2.env &&"
```

```suggestion
          HERMETO_SOURCE=". /cachi2/cachi2.env &&"
```

In the past, we ran into an issue where some container images had shells without `source`, only `.`